### ISSUE TITLE
Pin to specific `xmldom` commit

### ISSUE THREAD

**Issue by azasypkin** [auth0/node-samlp#65]
Currently `node-samlp` has the following dependency:

```json
"xmldom": "https://github.com/auth0/xmldom/tarball/master",
```

It doesn't look good to not pin to a specific commit. Today `auth0/xmldom` merged a commit and obviously `https://github.com/auth0/xmldom/tarball/master` points to a new code now without _any_ change in `node-samlp` version (+ yarn/npm lock files aren't happy either).

**Comment by mikeops** [comment]
Hi @azasypkin. Sorry for any issues that this caused you.

We've published a new version (3.3.3) that has `xmldom` commit pinned to a release tag. https://github.com/auth0/node-samlp/releases/tag/v3.3.3

**Comment by mikeops** [comment]
Closing this issue as the change has been made.
